terraform/aws/env/us-east/terraform.tfvars

```diff
@@ -1,7 +1,7 @@
-region            = "us-east-1"
-ami               = "ami-066a7f2ffac02f833"
-instance_type     = "t2.medium"
-key_name          = "KEY_NAME"
-server_count      = "3"
-client_count      = "4"
-
+region               = "us-east-1"
+ami                  = "ami-090a41df9e193a506"
+server_instance_type = "t2.medium"
+client_instance_type = "p3.2xlarge"
```

Comment:
Should the nvidia instance be in a different vars files? p3.2x is expensive but I don't have a sense of how often one needs it.

Reply:
i'll change these to more reasonable defaults